tests/edit/test_convert_null_if_not_in_list.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from chispa import assert_df_equality
+
+from cishouseholds.edit import convert_null_if_not_in_list
+
+
+@pytest.fixture
```

Comment:
Fixtures are only necessary when the same resource is shared by multiple tests - I'd move the expected and input data inside the test function here